src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -90,20 +90,20 @@ public void storeMessage(Message message) {
   List<Message> messages = new ArrayList<>();
 
   for (Entity entity : results.asIterable()) {
-   try {
-    String idString = entity.getKey().getName();
-    UUID id = UUID.fromString(idString);
-    String user = (String) entity.getProperty("user");
-    String text = (String) entity.getProperty("text");
-    long timestamp = (long) entity.getProperty("timestamp");
-
-    Message message = new Message(id, user, text, timestamp);
-    messages.add(message);
-   } catch (Exception e) {
-    System.err.println("Error reading message.");
-    System.err.println(entity.toString());
-    e.printStackTrace();
-   }
+     try {
```

Comment:
nit: Please remove the indentation changes. What you have here doesn't match the formatting in the rest of the file.